terraform/docker/Gemfile

```diff
@@ -10,6 +10,7 @@ source 'https://rubygems.org' do
   gem 'csv'
   gem 'mutex_m'
   gem 'syslog'
+  gem 'inspec-bin'
```

Comment:
This is not allowed due to license restrictions.

cinc-auditor-bin is the proper tool.